src/validation/validateRequiredFences.ts

```diff
@@ -0,0 +1,120 @@
+import reportError from '../core/reportError';
+import * as glob from 'glob';
+import * as path from 'path';
+import getAllConfigs from '../utils/getAllConfigs';
+import Config from '../types/config/Config';
+import ConfigSet from '../types/ConfigSet';
+
+interface ConfigPathPair {
```

Comment:
>ConfigPathPair [](start = 10, length = 14)

`Config` has it's path as one of its properties, so you shouldn't need this structure.